src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderAlert.kt

```diff
@@ -162,9 +162,10 @@ class OffenderAlertBuilder(
     commentText = commentText,
     verifiedFlag = verifiedFlag,
   ).apply {
-    workFlow = addWorkFlowLog(workActionCode = repository.lookupWorkFLowAction(WorkFlowAction.DATA_ENTRY))
+    addWorkFlowLog(workActionCode = repository.lookupWorkFLowAction(WorkFlowAction.DATA_ENTRY))
   }
     .let { repository.save(it) }
+    .also { workFlow = it.workFlows.first() }
```

Comment:
set the workFlow instance after it has been saved so the child logs referenced a saved entity - weird this worked before upgrade.

Reply:
That is weird. I always thought it was quite nice that Hibernate allowed this - didn't realise it was a bug!